content/en/metrics/volume.md

```diff
@@ -0,0 +1,120 @@
+
+To find the metrics not actively queried in the past 30 days, click on **Not Actively Queried** in the *Query Activity Facet* box. Selecting **Not Actively Queried** generates a list of unused metric names across dashboards, notebooks, monitors, SLOs, Metrics Explorer, and the API.
+
+## How to quickly reduce metric volume and cost
+
+After you identify unqueried metrics, you can quickly and confidently eliminate the volume and cost of these metric names by using [Metrics without Limits™][3] without a single line of code. By using Metrics without Limits, you ensure that you pay only for the metrics that you use by eliminating timeseries that are never or rarely leveraged. Based on our intelligent query insights, the average customer's custom metrics volume can be reduced by 70% if they were to use Metrics without Limits on their unqueried metric names. 
+
+To configure multiple unqueried metrics at once
+1. Click the **Configure Metrics** dropdown and select **Manage Tags** to open the [Metrics without Limits™ Tag configuration modal][4].
+2. Specify the metric namespace of the unqueried metrics you'd like to apply a bulk tag configuration to.
+3. Select **Include tags…** and set an empty allowlist of tags.
+
+{{< img src="metrics/volume/configure_metrics.png" alt="Configure Metric dropdown at the top of the page highlighting the Manage tags option" style="width:100%;" >}}
+
+You have full control over the cardinality of your metrics without the need to change your applications nor the requirement of a remote-write setup. Below is an example of how eliminating timeseries that are rarely used can significantly reduce your custom metrics volumes and costs.
+
+In this example, the tag configuration modal shows a metric with a current volume of 13690031 indexed custom metrics. After you select Include tags… with an empty allowlist of tags, the modal shows an estimated new volume of 1. You can reduce the number of indexed custom metrics by 13690030.
+
+{{< img src="metrics/volume/reduce_metric_vol_cost_tags.png" alt="Tag configuration modal showing an example metric with a current volume of 13690031 index metrics and an estimated new volume of 1, with an empty allowlist of tags" style="width:80%;" >}}
+
+## Analyze metrics' utility and relative value in Datadog
+As part of our Metrics without Limits suite of governance features, you can now quickly pinpoint valuable metrics that are underutilized in Datadog with the Metrics Related Assets feature. A metrics related asset refers to any dashboard, notebook, monitor or SLO that queries a particular metric. Our intelligent query insights surface the popularity of these related assets as well as the quantity so you can evaluate metric utility within your organization, enabling data-driven decisions. This feature allows you to identify how your team can utilize existing metrics to get more value from your observability spend and [reduce metric volume and cost].
```

Comment:
Did you mean to link to another document here? `[reduce metric volume and cost]`

```suggestion
Metrics without Limits™ allows you to quickly find metrics that are underused in Datadog with the Metrics Related Assets feature. A metrics related asset refers to any dashboard, notebook, monitor, or SLO that queries a particular metric. Datadog's intelligent query insights surface the popularity and quantity of these related assets so you can evaluate metric utility within your organization. Use this information to make data-driven decisions. Identify how your team can use existing metrics to get more value from your observability spend and reduce metric volume and cost.
```

Reply:
I think that was actually one of your original edits? maybe it was meant to link to the later [reduce metric volume cost] section in this same page?